### ISSUE TITLE
Rename exported assets to a descriptive name

### ISSUE THREAD

**Issue by MackenzieShirk** [Raider-Arts/painter-megascan-link#8]
![image](https://user-images.githubusercontent.com/9064885/135717910-aade5320-13af-4e2b-854c-0d4858923939.png)

Currently this is an issue with Megascans, and I recall Quixel mentioning that they cant give the textures descriptive names due to how they index the file names. It would be extremely helpful to mitigate this when exporting to Painter by either using the name given in Bridge, or coming up with some other naming scheme. As it is, importing several textures will create extremely unreadable filenames in the Shelf. 

![image](https://user-images.githubusercontent.com/9064885/135717966-c6fa1719-c5ab-4ab1-902a-a47e164c8336.png)

**Comment by faggionluca** [comment]
Yeah i can look into this and see if i can come up with a better naming scheme or extract somehow the bridge display name for the textures

**Comment by faggionluca** [comment]
@MackenzieShirk i've tried to implement this in the plugin but the current JS API of Substance Painter uses the file path as the display name for imported assets so the only way to implement this kind of feature would be to duplicate the assets in the file system rename them using the configured naming scheme of Bridge and then import them into Painter, this method is far from ideal since it will require write and read access of the file system and thus causing not permitted errors
